pages/api/id/[form]/retrieval.ts

```diff
@@ -46,6 +46,34 @@ function connectToDynamo(): DynamoDBDocumentClient {
   return DynamoDBDocumentClient.from(db);
 }
 
+interface dynamoDBItem {
```

Comment:
This interface is not correct. We're using the DynamoDB document client here which allows us to work with native JS types